scripts/evaluate_model.py

```
@@ -0,0 +1,561 @@
+                result.timestamp_notes = "adapt_nemo_hypotheses succeeded"
+
+                for fmt in formats:
+                    spec = FORMATTERS.get(fmt)
+                    if spec is None:
+                        continue
+                    try:
+                        text = spec.format_func(aligned)
+                        out_path = output_dir / f"{audio_path.stem}_{_slugify(model_name)}.{fmt}"
+                        out_path.write_text(text, encoding="utf-8")
+                        result.output_paths[fmt] = str(out_path)
+                    except Exception as fmt_exc:  # pragma: no cover
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)